AsyncDisplayKit/ASTextNode.mm

```diff
@@ -250,9 +248,28 @@ - (ASTextKitAttributes)_rendererAttributes
   };
```

Comment:
I added a lock for it for consistency of this PR now, but I agree and as @ocrickard said in his comment in the future we should introduce a naming for methods that must be called with the lock held?